app/src/App.tsx

```diff
@@ -6,15 +6,19 @@ import { ThemeProvider } from './components/theme-provider';
 import { Toaster } from './components/ui/toaster';
 import { AuthManager } from './lib/auth';
 import Routes from './routes';
+import { useRouteManager } from './routes/Provider';
+import { Route } from './lib/routes';
 
 const AuthContext = createContext(new AuthManager());
 export const useAuthManager = () => useContext(AuthContext);
 
 function App() {
+    const routeManager = useRouteManager();
+
     return (
         <ThemeProvider defaultTheme="dark" storageKey="vite-ui-theme">
             <TooltipProvider>
-                <AppBar />
+                {routeManager.route !== Route.LOGIN && <AppBar />}
```

Comment:
No App Bar means no drag... Could make it black for the occasion?

Reply:
No drag? For the rest of the app or what do you mean? I don't quite understand.

Reply:
Try dragging the window without the AppBar. You won't be able to drag it if its disabled. So instead, we can make it transparent or black when route is Login. You don't have to do this, I can update it once we agree on an implementation.

Reply:
Actually, I updated with what I mean